dali/benchmark/dali_bench.h

```diff
@@ -33,11 +33,11 @@ class DALIBenchmark : public benchmark::Fixture {
  public:
   DALIBenchmark() {
     rand_gen_.seed(time(nullptr));
-    LoadJPEGS(image_folder, &jpeg_names_, &jpegs_, &jpeg_sizes_);
+    LoadJPEGS(image_folder, &jpeg_names_, &jpegs_);
   }
 
   virtual ~DALIBenchmark() {
-    for (auto &ptr : jpegs_) {
+    for (auto &ptr : jpegs_.data_) {
```

Comment:
Is it more future-proof to delete jpegs_ and count on its destructor to delete jpegs_.data_ in turn?  That would mean `ImgSetDescr` in dali/util/image.h should be a class rather than a struct and have a non-default destructor (which is probably reasonable anyway).

Reply:
Yes, it is a very useful change. It also fixes memory leak in DALISingleOpTest